conductor/lib/uploader_v2.py

```diff
@@ -281,15 +280,34 @@ def __init__(self,
         self.last_touch = datetime.datetime.now()
         self._results_queue = results_queue
 
+    def log(self, level, message, fields={}):
```

Comment:
Perhaps the `level` argument could be an optional argument with a default value of `logging.INFO` (or whatever level would be intuitive).